hedera-node/src/main/java/com/hedera/services/grpc/GrpcStarter.java

```diff
@@ -92,8 +95,8 @@ void logInfoWithConsoleEcho(String s) {
 	}
 
 	private boolean thisNodeIsDefaultListener() {
-		final var thisAccount = nodeAddress.getMemo();
 		final var blessedNodeAccount = nodeLocalProperties.devListeningAccount();
-		return blessedNodeAccount.equals(thisAccount);
+		final var staticBook = addressBook.get();
+		return blessedNodeAccount.equals(staticBook.getAddress(nodeId.getId()).getMemo());
```

Comment:
@tinker-michaelj aren't we going to deprecate usage of memo field from the NodeAddress proto? Would this need a refactor when we do that ??

Reply:
The `AddressBook` object here doesn't come from a protobuf message, but from _config.txt_; and until Platform changes, the memos there will always contain account ids.

Reply:
Got it. The protobuf message is built from this config.txt anyway right.